pennylane_honeywell/device.py

```diff
@@ -76,7 +78,7 @@ class HQSDevice(QubitDevice):
     """
     # pylint: disable=too-many-instance-attributes
     name = "Honeywell Quantum Solutions PennyLane plugin"
-    pennylane_requires = ">=0.9.0"
+    pennylane_requires = ">=0.11.0"
```

Comment:
@mariaschuld I don't think this is needed here; the Honeywell plugin will continue to work with v0.9 and v0.10 of PennyLane since there have been no breaking changes here.

Reply:
We generally only update this if the plugin _requires_ the latest PennyLane to work. Otherwise, being too strict can cause dependency issues downstream for the user.

Reply:
Although, I think this is the _only_ plugin that doesn't need to have `pennylane_requires` incremented 😆

Reply:
Yes, should be :)